.changeset/ten-mirrors-repair.md

```diff
@@ -0,0 +1,7 @@
+---
+'ember-headless-form': patch
+---
+
+Use describedby instead of errormessage ARIA attribute
+
+Support for `aria-errormessage` is [very incomplete across screen readers](https://a11ysupport.io/tech/aria/aria-errormessage_attribute), therefore switching to the [better supported](https://a11ysupport.io/tech/aria/aria-describedby_attribute), but less specific `aria-describedby`.
```

Comment:
Thanks for this detailed explanation! Really helps a lot!

Reply:
credits belong to @ynotdraw, who found out about this in https://github.com/CrowdStrike/ember-toucan-core/pull/27#discussion_r1105952518